thirdeye/thirdeye-frontend/app/pods/home/route.js

```diff
@@ -96,13 +96,14 @@ export default Route.extend({
    * Sets the table column, metricList, and alertList
    * @return {undefined}
    */
-  setupController(controller) {
+  setupController(controller, model) {
     this._super(...arguments);
 
     controller.setProperties({
       columns,
       metricList: this.getMetrics(),
-      alertList: this.getAlerts()
+      alertList: this.getAlerts(),
+      defaultApplication: model.applications[0]
```

Comment:
perfect! As controller itself being a singleton (it does not get called over and over like the route) is not ideal for setting anything dynamic and having a need of model itself. The `setupController` is primarily for setting up some controller context dynamically. And in this case, it is a good usage of   accessing the model and setting the controller's `defaultApplication` property.  Thanks for doing this!